router/network.go

```diff
@@ -19,108 +19,16 @@ type NetHandleT struct {
 	httpClient *http.Client
 }
 
-func (network *NetHandleT) processOldResponseType(jsonData []byte) (int, string, string) {
+func (network *NetHandleT) sendPost(jsonData []byte) (int, string, string) {
```

Comment:
Please add a comment describing what this function returns, because it returns 3 things and 2 of the same type. Maybe even named returns. But I am not a big fan of bare return statements. Even though you are using named returns, I would still prefer return statement with parameters included.